qiskit_ibm_runtime/visualization/utils.py

```diff
@@ -0,0 +1,855 @@
+            [2, 4],
+            [2, 5],
+            [2, 6],
+            [2, 7],
+            [2, 8],
+            [3, 0],
+            [3, 4],
+            [3, 8],
+            [4, 0],
+            [4, 1],
+            [4, 2],
+            [4, 3],
+            [4, 4],
+            [4, 5],
+            [4, 6],
+            [4, 7],
+            [4, 8],
+        ]
+
+    if num_qubits == 53:
+        return [
+            [0, 2],
+            [0, 3],
+            [0, 4],
+            [0, 5],
+            [0, 6],
+            [1, 2],
+            [1, 6],
+            [2, 0],
+            [2, 1],
+            [2, 2],
+            [2, 3],
+            [2, 4],
+            [2, 5],
+            [2, 6],
+            [2, 7],
+            [2, 8],
+            [3, 0],
+            [3, 4],
+            [3, 8],
+            [4, 0],
+            [4, 1],
+            [4, 2],
+            [4, 3],
+            [4, 4],
+            [4, 5],
+            [4, 6],
+            [4, 7],
+            [4, 8],
+            [5, 2],
+            [5, 6],
+            [6, 0],
+            [6, 1],
+            [6, 2],
+            [6, 3],
+            [6, 4],
+            [6, 5],
+            [6, 6],
+            [6, 7],
+            [6, 8],
+            [7, 0],
+            [7, 4],
+            [7, 8],
+            [8, 0],
+            [8, 1],
+            [8, 2],
+            [8, 3],
+            [8, 4],
+            [8, 5],
+            [8, 6],
+            [8, 7],
+            [8, 8],
+            [9, 2],
+            [9, 6],
+        ]
+
+    if num_qubits == 65:
+        return [
+            [0, 0],
+            [0, 1],
+            [0, 2],
+            [0, 3],
+            [0, 4],
+            [0, 5],
+            [0, 6],
+            [0, 7],
+            [0, 8],
+            [0, 9],
+            [1, 0],
+            [1, 4],
+            [1, 8],
+            [2, 0],
+            [2, 1],
+            [2, 2],
+            [2, 3],
+            [2, 4],
+            [2, 5],
+            [2, 6],
+            [2, 7],
+            [2, 8],
+            [2, 9],
+            [2, 10],
+            [3, 2],
+            [3, 6],
+            [3, 10],
+            [4, 0],
+            [4, 1],
+            [4, 2],
+            [4, 3],
+            [4, 4],
+            [4, 5],
+            [4, 6],
+            [4, 7],
+            [4, 8],
+            [4, 9],
+            [4, 10],
+            [5, 0],
+            [5, 4],
+            [5, 8],
+            [6, 0],
+            [6, 1],
+            [6, 2],
+            [6, 3],
+            [6, 4],
+            [6, 5],
+            [6, 6],
+            [6, 7],
+            [6, 8],
+            [6, 9],
+            [6, 10],
+            [7, 2],
+            [7, 6],
+            [7, 10],
+            [8, 1],
+            [8, 2],
+            [8, 3],
+            [8, 4],
+            [8, 5],
+            [8, 6],
+            [8, 7],
+            [8, 8],
+            [8, 9],
+            [8, 10],
+        ]
+
+    if num_qubits == 127:
```

Comment:
Looks like this is missing some herons, like 133.

Reply:
There's nowhere I can copy/paste them from. Is it okay if I add support for 133 Heron when I refactor the coordinates?

Reply:
sure